lib/chef/provider/chef_organization.rb

```diff
@@ -52,12 +52,25 @@ def whyrun_supported?
     new_resource.members.each do |user|
       if !existing_members.include?(user)
         converge_by "Add #{user} to organization #{new_resource.name}" do
-          rest.post("#{rest.root_url}/organizations/#{new_resource.name}/users/#{user}", {})
+          add_user_to_org(user, new_resource.name)
         end
       end
     end
   end
 
+  def add_user_to_org(user, org)
+    response = rest.post("#{rest.root_url}/organizations/#{org}/association_requests", { 'user' => user })
```

Comment:
Can you add a `# TODO` comment here saying we would like to use the single API but it wasn't working?